### DISCUSSION TITLE
Template Mobile

### DISCUSSION THREAD

**Discussion by hitkoch** [YetiForceCompany/YetiForceCRM#17494] [Wishlist]
Is there a template for mobile?
Does anyone have one to sell?
Does anyone know where I can buy it?

**Comment by paula-w** [comment]
There isn't. 
The system is highly responsive though and can be used on your mobile devices. You can test it by opening https://demo.yetiforce.com/ in the browser on your phone/tablet.

**Reply by JohnMaverick** [reply]
A bit OT... will YF 7 code be available in repo?

**Comment by hitkoch** [comment]
Hi Paula, I ran several responsive tests, but many resources do not have attractive usability, for example when launching a sale, when we enter products and discounts, the products resource is not responsive, notice in the image that it is not possible to view the fields for apply discounts and taxes.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/22005797/9183aeef-4b73-4f16-8bf7-5e621f3baac3)
